app/lib/pdf_filler/ny_it2_pdf.rb

```diff
@@ -0,0 +1,60 @@
+module PdfFiller
+  class NyIt2Pdf
+    include PdfHelper
+
+    def source_pdf_name
+      "it2-TY2023"
+    end
+
+    def initialize(submission, kwargs)
+      @submission = submission
+      @kwargs = kwargs
+    end
+
+    def hash_for_pdf
+      w2 = @kwargs[:w2]
+      answers = {
+        'Box_a' => w2.employee_ssn,
+        'Box_b' => w2.employer_ein,
+        'Box_c1' => w2.employer_name,
+        'Box_c2' => w2.employer_street_address,
+        'Box_c3_city' => w2.employer_city,
+        'Box_c3_state' => w2.employer_state,
+        'Box_c3_zip' => w2.employer_zip_code,
+        'Box_c3_county' => 'USA', #TODO check if other countries allowed
```

Comment:
This is county, not country

Reply:
the pdf field name is county when i dump the data fields. But the label on the pdf outside the box is Country